### PR TITLE
fix: download after template, move download & colab under each left tab

### PR DESCRIPTION
**Pull request by ydcjeff**
### Description

ref: https://github.com/pytorch-ignite/code-generator/issues/103#issuecomment-859069323

### Additional context

---

### What is the purpose of this pull request?

- [x] Bug fix
- [ ] New feature
- [ ] Other

### CODE REVIEW

**Comment by vfdev-5** [comment]
@ydcjeff let's move download and colab somewhere else instead of removing others.

**Comment by ydcjeff** [comment]
Where shall we move?

**Comment by vfdev-5** [comment]
For example, in the end of left widget

**Comment by vfdev-5** [comment]
@ydcjeff this is nicer ! How about splitting left widget into 2 such "download" and "open in colab" are visible from any tab : templates, training, handlers, loggers ?

**Comment by trsvchn** [comment]
yes, can we anchor them?

**Comment by ydcjeff** [comment]
> @ydcjeff this is nicer ! How about splitting left widget into 2 such "download" and "open in colab" are visible from any tab : templates, training, handlers, loggers ?

I have tried to appear them in any tab. But contents in handler tab are long and feels not having enough space.

File: src/components/NavBar.vue [hunk 1]
```diff
@@ -13,8 +13,8 @@
     </h1>
     <div class="left-side-badges">
       <NavHelp />
-      <NavDownload />
-      <NavColab />
+      <!-- <NavDownload /> -->
+      <!-- <NavColab /> -->
```
Comment (ydcjeff) [inline src/components/NavBar.vue:17]:
```suggestion
```

**Comment by trsvchn** [comment]
Ok, looks not that nice. What about anchoring on the top, because tabs have different "height" and buttons float out of the viewing field

**Comment by trsvchn** [comment]
It is definitely better on the top, but not sure if it is the perfect place :(

**Pull request merged**